index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Abdul Ali Noori's portfolio" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <section>
+        <nav class="top-navigation">
+            <div class="welcome-message">Welcome</div>
+            <img src="images/Icons/Icon-Menu.svg" alt="GitHub">
+        </nav>
```

Comment:
- [ ] Please wrap the nav tag in (`header `tags ) since typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. Check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 
- [ ] The (`header` tags) should only wrap around the toolbar section. ie. The logo and the hamburger. :+1: